frame/evm/precompile/modexp/src/lib.rs

```diff
@@ -198,7 +201,7 @@ impl Precompile for Modexp {
 				exit_status: ExitSucceed::Returned,
 				output: bytes.to_vec(),
 			})
-		} else if bytes.len() < mod_len {
+		} else if dbg!(bytes.len()) < mod_len {
```

Comment:
why `dbg!` here?

Reply:
Forgot to remove it. When `mod_len` is 0, the precompile must return empty data. However `bytes.len()` returns at least 1 (which I saw using `dbg!` ;)), which made the precompile return an error. This case is now handled earlier like in geth code.